scripts/enforce-change-notes-updated-on-pr-to-master

```diff
@@ -10,8 +10,8 @@ change_notes_path=CHANGE-NOTES.md
 if [[ ${CI_PULL_REQUEST-} ]]; then
   # In case of a PR build, CI_PULL_REQUEST should be a link of the form https://github.com/VirtusLab/git-machete-intellij-plugin/pull/123
   pr_num=${CI_PULL_REQUEST##*/}
-  # Implicit usage of ${GITHUB_TOKEN} by hub
-  base_branch=$(hub pr show --format=%B "$pr_num")
+  # Implicit usage of ${GITHUB_TOKEN} by gh
+  base_branch=$(gh pr view -w "$pr_num") #the gh command does not have an equivalent option for --format
```

Comment:
`--json` and/or `--jq`

Reply:
but we don't need the `-w`  tag because we just want to extract the base branch name. we don't intend to open the PR in the browser right?

Reply:
Whoops yes, `-w` is definitely not expected in CI

Reply:
I mean, although Selenium tests in certain apps do run a browser in Docker/CI ;D but that's not our case